packages/playwright/types/testReporter.d.ts

```diff
@@ -567,7 +567,7 @@ export interface JSONReportTestStep {
   steps?: JSONReportTestStep[];
 }
 
-export type JSONReportSTDIOEntry = { text: string } | { buffer: string };
```

Comment:
note: either revert or keep this. Since we collect the stdio from the parent process now with `setEncoding('utf8')` it will never be a buffer anymore.